extensions/functions_string.yaml

```diff
@@ -51,6 +51,12 @@ scalar_functions:
     description: >-
       Extract a substring of a specified `length` starting from position `start`.
       A `start` value of 1 refers to the first characters of the string.
+
+      The `negative_start` option applies to the `start` parameter. `WRAP_FROM_END` means
+      the returned substring will start from the end of the `input`. The last character
+      has an index of -1.  `LEFT_OF_BEGINNING` means the returned substring will start from
```

Comment:
This description of `WRAP_FROM_END` seems a little confusing.

> the returned substring will start from the end of `input`.

The returned substring won't start at the end right?  For example, substr("hello", -3, 1) returns "l" which doesn't start at the end of the "hello"

> The last character has an index of -1

This is correct, but it is not clear to me that we count backwards, in other words, it is not clear that -2 is the second from last character.